source/templates.tex

```diff
@@ -1375,11 +1377,7 @@
 \tcode{float},
 and list of pointers to
 \tcode{Shape}s.
```

Comment:
I think it's lacking something -- I think we want to more explicitly say that a single `List` template can simultaneously provide definitions for lists of type `T`, for all possible types `T`.